### PR TITLE
Update removeAdUnit

### PR DESCRIPTION
**Pull request by deekshithraop**
## Type of change

- [x] Feature

### pbjs.removeAdUnit()

updated removeAdUnit so that if it is invoked with no arguments it will remove all add units, and it will also accept an array argument of addUnitCodes and will only remove them.

- link to a PR on the docs change(https://github.com/prebid/prebid.github.io/pull/1138)

### CODE REVIEW

**Comment by bretg** [comment]
What's the use case here?

**Comment by mkendall07** [comment]
modification of adUnits in single page applications. Eventually we should remove `pbjs.adUnits` as a globally accessible property,.

File: src/prebid.js [hunk 1]
```diff
@@ -375,6 +375,16 @@ $$PREBID_GLOBAL$$.removeAdUnit = function (adUnitCode) {
   }
 };
 
+/**
+ * Remove all adUnits from the $$PREBID_GLOBAL$$ configuration
+ * @alias module:pbjs.removeAllAdUnits
+ */
+$$PREBID_GLOBAL$$.removeAllAdUnits = function () {
```
Comment (mkendall07) [inline src/prebid.js]: hmmm should we make this `removeAdUnits([adUnitCodeArray])` ? seems more flexible.

**Comment by deekshithraop** [comment]
@mkendall07 made changes and updated description

File: src/prebid.js [hunk 2]
```diff
@@ -360,19 +360,33 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id) {
...
 $$PREBID_GLOBAL$$.removeAdUnit = function (adUnitCode) {
   utils.logInfo('Invoking $$PREBID_GLOBAL$$.removeAdUnit', arguments);
-  if (adUnitCode) {
+
+  if (!adUnitCode) {
+    $$PREBID_GLOBAL$$.adUnits = [];
+    return;
+  }
+
+  var adUnitCodes;
```
Comment (mkendall07) [inline src/prebid.js]: prefer let.

**Comment by mkendall07** [comment]
@bretg fyi we decided to not add an extra method here but instead reuse the existing one.

File: src/prebid.js (hunk 2 above)
Comment (deekshithraop) [reply src/prebid.js]: done

**Pull request merged**